scylla-server/tests/data-services.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, test, expect } from 'vitest';
+import { getDataByDataTypeName } from '../src/services/data.services';
+
+/**
+ * Unit Test for Get All Data by DataType Name
+ * Checks that given datatype name, functionr returns expected data
+ * Currently no data in database
+ */
+describe('Data', () => {
+  test('Get All Data by DataType Name works w valid data', async () => {
+    const expected = [];
+    const data = JSON.parse('{"dataTypeName": "test"}');
```

Comment:
why does this work?

Reply:
since data is of right typing, function does not throw error and since there is no data with dataTypeName "test" in the db it returns an empty list

Reply:
oh right w